kustomize/commands/localize/localize.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localize
+
+import (
+	"fmt"
+	"io"
+	"log"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+	lclzr "sigs.k8s.io/kustomize/api/krusty/localizer"
+	"sigs.k8s.io/kustomize/kyaml/errors"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+const cmdName = "localize"
+const numArgs = 2
+const alphaWarning = `Warning: This is currently an alpha command.
+`
+
+type arguments struct {
+	target string
+	dest   string
+}
+
+type flags struct {
+	scope string
+}
+
+// NewCmdLocalize returns a new localize command.
+func NewCmdLocalize(fs filesys.FileSystem, writer io.Writer) *cobra.Command {
+	log.SetOutput(writer)
+	var f flags
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s [target [destination]]", cmdName),
+		Short: "Creates localized copy of target kustomization root at destination",
+		Long: fmt.Sprintf(`Creates copy of target kustomization directory
+or versioned URL at destination, where remote references in the original are 
+replaced by local references to the downloaded remote content.
+
+If target is not specified, the current working directory will be used. 
+Destination is a path to a new directory in an existing directory. If 
+destination is not specified, a new directory will be created in the current 
+working directory. 
+
+For details, see: https://kubectl.docs.kubernetes.io/references/kustomize/cmd/
+
+Disclaimer:
+This command does not yet localize helm or KRM plugin fields. This command also
+alphabetizes kustomization fields in the localized copy.
+
+%s`, alphaWarning),
+		Example: fmt.Sprintf(`
+# Localize the current working directory, with default scope and destination
+%s %s 
```

Comment:
With all our other commands, we write out the full command in the example (so in this case, `kustomize localize`), instead of using %s placeholders. It is a little cleaner and easier to follow. It is very unlikely that these names will ever change, and if they do, updating the examples will be the least of our worries.

Reply:
Done. I replaced the placeholders with the written string.